src/components/DataCollector/index.js

```diff
@@ -25,11 +25,11 @@ const createDataCollector = ({ eventManager, logger }) => {
           let { xdm } = options;
           const {
             data,
-            viewStart = false,
             documentUnloading = false,
             type,
             mergeId,
-            scopes = []
+            renderDecisions = false,
+            decisionsScopes = []
```

Comment:
Curious to know how others feel about this name. My initial reaction is that `decisionScopes` feels better than `decisionsScopes`.

Reply:
@Aaronius  I like `decisionScopes` because I can pronounce it ;)

Reply:
@ninaceban Mind changing it to `decisionScopes`? I think grammatically it probably makes more sense and @jfkhoury and I will be able to pronounce it. 😄

Reply:
ok, I'll change that. thank you